vtkext/private/module/vtkF3DWGLRenderWindow.cxx

```diff
@@ -0,0 +1,102 @@
+#include <vtkObjectFactory.h>
+
+#include "vtkF3DWGLRenderWindow.h"
+
+#include <Windows.h>
+#include <dwmapi.h>
+
+#ifndef DWMWA_USE_IMMERSIVE_DARK_MODE
+#define DWMWA_USE_IMMERSIVE_DARK_MODE 20
+#endif
+
+constexpr auto IMMERSIVE_DARK_MODE_SUPPORTED_SINCE = 19041;
+
+vtkStandardNewMacro(vtkF3DWGLRenderWindow);
```

Comment:
standard new macro are usually put after internal unamed namespace

Reply:
will move it